### DISCUSSION TITLE
Cleanup suggestion - Unused `.png` icons

### DISCUSSION THREAD

**Discussion by adevade** [statamic/cms#6000] [General]
Found some `.png` icons that doesn't seem to be used anymore (with a quick search): [/resources/img](https://github.com/statamic/cms/tree/e73f68cf74dd2c76723f23520d474fc4b73fa634/resources/img)

Some are being used though, and could probably be replaced with `.svg` versions: [calendar.png](https://github.com/statamic/cms/blob/e73f68cf74dd2c76723f23520d474fc4b73fa634/resources/sass/components/date-picker.scss#L38)

The others could probably be removed from the repo?

**Comment by edalzell** [comment]
PRs are gladly accepted!

**Reply by adevade** [reply]
Opened a PR here: https://github.com/statamic/cms/pull/6006
